sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusExtensionConfigProvider.cs

```diff
@@ -112,7 +112,11 @@ public void Initialize(ExtensionConfigContext context)
 
         internal static ParameterBindingData ConvertReceivedMessageToBindingData(ServiceBusReceivedMessage message)
         {
-            ReadOnlyMemory<byte> messageBytes = message.GetRawAmqpMessage().ToBytes().ToMemory();
+// TEMP (target scale release):
```

Comment:
Just wanted to ensure this was intentional.

Reply:
Yes, unfortunately.  The out-of-band request has us dodging dependency changes that we don't want to release yet.   So, we're temporarily suppressing new (unused functionality) until after release.